.github/workflows/go.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        go-version: [1.19, "1.20"]
+        go-version: ["1.21", "1.22", "1.23"]
```

Comment:
We want to support Go 1.20 which is also supported by AWS SDK. Please change the min to 1.20 in the PR.

Reply:
Was looking at outdated documentation, the library supports 1.21 as the minimum version. Approved.